inventory/idm/hosts

```diff
@@ -0,0 +1,7 @@
+[idm]
+
+[dns-server]
+localhost ansible_connection=local
```

Comment:
can we please add a `host_vars/localhost.yml` for the `ansible_connection: local` content?